policy_packs/gcp/kms/check_crypto_keys_are_rotated_regularly/README.md

```diff
@@ -3,7 +3,7 @@ categories: ["data protection", "security"]
 primary_category: "data protection"
 ---
 
-# Enforce GCP KMS Crypto Keys to be rotated on regular basis
+# Check If GCP KMS Crypto Keys Are Rotated Regularly
```

Comment:
Pleaes check if the Policy Pack AKA matches the title here.

Reply:
yes it matches > `gcp_kms_check_crypto_keys_are_rotated_regularly`